### PR TITLE
Changing Workshop Documentation.

### PR DESCRIPTION
**Pull request by brahmaroutu**

### CODE REVIEW

File: workshop/README.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ IBM Cloud provides the capability to run applications in containers on Kubernete
 
 [Lab 0](Lab0) (Optional): Provides a walkthrough for installing IBM Cloud command-line tools and the Kubernetes CLI. You can skip this lab if you have the IBM Cloud CLI, the container-service plugin, the containers-registry plugin, and the kubectl CLI already installed on your machine.
 
-[Lab 1](Lab1): This lab walks through creating and deploying a simple "hello world" app in Node.JS, then accessing that app.
+[Lab 1](Lab1): This lab walks through creating and deploying a simple "guestbook" app written in Go as a net/http Server([negroni](https://github.com/codegangsta/negroni) based) then accessing that app.
```
Comment (MHBauer) [inline workshop/README.md]: we don't need this link

File: workshop/Lab2/README.md [hunk 2]
```diff
@@ -71,30 +71,30 @@ A *replica* is how Kubernetes accomplishes scaling out a deployment. A replica i
...
 
 To update and roll back:
-1. Build the new docker image with a `2` tag:
+1. Pull the new docker image with a `v2` tag:
 
-   ```docker build --tag registry.ng.bluemix.net/<my_namespace>/hello-world:2 .```
+   ```docker pull ibmcom/guestbook:v2```
 
-2. Push the image to the IBM Cloud Container Registry:
+2. Push the image, optionally, to the IBM Cloud Container Registry:
```
Comment (MHBauer) [inline workshop/Lab2/README.md]: Do not optionally do anything. Do it or do not. What's in the script we run?

File: workshop/Lab2/README.md [hunk 3]
```diff
@@ -71,30 +71,30 @@ A *replica* is how Kubernetes accomplishes scaling out a deployment. A replica i
 
 Kubernetes allows you to use a rollout to update an app deployment with a new Docker image.  This allows you to easily update the running image and also allows you to easily undo a rollout, if a problem is discovered after deployment.
 
-In the previous lab, we created an image with a `1` tag. Let's make a version of the image that includes new content and use a `2` tag. This lab also contains a `Dockerfile`. Let's build and push it up to our image registry.
+In the previous lab, we used an image with a `v1` tag. Let's make a version of the image that includes new content and use a `v2` tag. Let's pull the image from public registry and optionally push it up to our image registry.
```
Comment (MHBauer) [inline workshop/Lab2/README.md]: no options

File: workshop/Lab2/README.md (hunk 2 above)
Comment (brahmaroutu) [reply workshop/Lab2/README.md]: The reason is that currently the only place where we showcase containerRegistry. There is no need for it and we can use the ibmcom image directly when deploy the Pod. thoughts?

File: workshop/Lab2/README.md [hunk 4]
```diff
@@ -51,16 +51,16 @@ A *replica* is how Kubernetes accomplishes scaling out a deployment. A replica i
    ```
    => kubectl get pods
    NAME                          READY     STATUS    RESTARTS   AGE
```
Comment (duglin) [inline workshop/Lab2/README.md]: Now the header doesn't line up with the columns

File: workshop/Lab2/README.md [hunk 5]
```diff
@@ -127,28 +123,28 @@ To update and roll back:
    Waiting for rollout to finish: 9 of 10 updated replicas are available...
    Waiting for rollout to finish: 9 of 10 updated replicas are available...
    Waiting for rollout to finish: 9 of 10 updated replicas are available...
-   deployment "hello-world" successfully rolled out
+   deployment "guestbook" successfully rolled out
    ```
 
    ```
    => kubectl get replicasets
    NAME                   DESIRED   CURRENT   READY     AGE
```
Comment (duglin) [inline workshop/Lab2/README.md]: columns are off here too

File: workshop/README.md (hunk 1 above)
Comment (duglin) [reply workshop/README.md]: +1 people won't care what http package we're using

File: workshop/Lab1/README.md [hunk 6]
```diff
@@ -45,46 +45,42 @@ which your cluster has access.
...
+   
    This action will take a bit of time. To check the status of your deployment, you can use `kubectl get pods`.
 
    You should see output similar to the following:
    
    ```
    => kubectl get pods
    NAME                          READY     STATUS              RESTARTS   AGE
-   hello-world-562211614-0g2kd   0/1       ContainerCreating   0          1m
+   guestbook-59bd679fdc-bxdg7   0/1       ContainerCreating   0          1m
```
Comment (duglin) [inline workshop/Lab1/README.md]: please align this

**Pull request merged**